python/tritonserver/_c/tritonserver_pybind.cc

```diff
@@ -1953,7 +1962,8 @@ PYBIND11_MODULE(triton_bindings, m)
           &PyInferenceRequest::RemoveAllRequestedOutputs)
       .def("set_string_parameter", &PyInferenceRequest::SetStringParameter)
       .def("set_int_parameter", &PyInferenceRequest::SetIntParameter)
-      .def("set_bool_parameter", &PyInferenceRequest::SetBoolParameter);
+      .def("set_bool_parameter", &PyInferenceRequest::SetBoolParameter)
+      .def("cancel", &PyInferenceRequest::Cancel);
```

Comment:
Would be good to test `cancel()` in follow-up if exposing it

Reply:
will add post merge as seperate PR